test/prepareAssetsPackage-test.js

```diff
@@ -66,7 +67,7 @@ it('picks out the right files', async () => {
   const zip = new AdmZip(buffer);
   expect(zip.getEntries().map(({ entryName }) => entryName)).toEqual([
     '1x1.jpg', // this is in the root because the css is always served from the root on happo workers
-    'one.jpg',
     'inlineResources/1x1.png',
+    'one.jpg',
```

Comment:
The order changes here because we are now sorting alphabetically by path at the end, and o is after i.